.tests/alexa.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-export LC_NUMERIC=en_US.UTF-8
+export LC_ALL=C.UTF-8
 format_number () {
   numfmt --grouping "$@"
 }
```

Comment:
I found a solution that works in `bash` and does not rely on `numfmt` or locales; it uses a hard-coded `,` and groups of three, which is probably what we want:
```
format_number () {
    [ "${#1}" -gt 3 ] && printf "$(format_number ${1:0:-3} ),${1: -3:3}" || printf "$1"
}
```
(The space in `${1: -3:3}` is necessary, just in case you're wondering.)

Reply:
Alright, I just added that.